src/app/layout.tsx

```diff
@@ -27,7 +28,7 @@ export default function RootLayout ({
 }): any {
   return (
     <html lang='en' className='snap-proximity snap-y'>
-      <body>
+      <body className={ inter.className}>
```

Comment:
As discussed we will deal with font change in a separate task. In summary can you limit the PR to the following:
1. Update Map icon
2. Change share link label (not icon)

Thanks!